keypom/keypom-airfoil
# Added claim calls and verification to claim pages

# Description
- Created a claim processing page to determine validity of the drop and redirect user to claim drop pages
- Token, NFT, Ticket claims

# How to test
1. check out locally and go to http://localhost:3000
2. Click on any of the linkdrops

NFT
  'http://localhost:3000/claim/v1-4.keypom.testnet/4qGXq1pM7hvUrScafrL694Nxk6V5t67uJGbW6pAYLUYg7LS76R4spw6pPhq6sj8zVFF8XEJbY2aZLKUjqJLJUmEW',
  'http://localhost:3000/claim/v1-4.keypom.testnet/44HTrVEfRCDfHx7jTYYYdcCdVG3aDYfChZ9RgXwHzzd2poLje8D7pHhYzXHG3vEvbf1yPPkCNqxW7atjMyQA15yH'
  
Token
'http://localhost:3000/claim/v1-4.keypom.testnet/3umzu1byobE9gmRzaiR4bvSyjcnaAioNrt1pqoRcz1VkGP99hnt4MBMnY9rixjVaFgrzujZr6QnQALPULCTqTyDZ',
  'http://localhost:3000/claim/v1-4.keypom.testnet/3fgVbWsvQ1k775MyGkdsy5c8EnqMCLjea1xRbxnG76Rb2YfsJC8US2C7d4zBQtyK789Q6oNVW7oYR5mHCGCdxjXL'
  
Ticket
 'http://localhost:3000/claim/v1-4.keypom.testnet/4Nj9Ah9A56FcnZc8msoKqN11QhaXJs7rxZ82cH6SDKrqN2Lq7jsxRgTi4f6cWZyzErj8zebWW1sxVjgN72KxauDN',
  'http://localhost:3000/claim/v1-4.keypom.testnet/5t5coLAXYZjeCbkzUjtWpFDRFi8kqn8UwvkyeyNa6SCrNeBbYxEw3DE1FmZiSUNs9CQUWr6hk3q3UvGk9ULkrN32'
  
3. For scanning ticket, go to http://localhost:3000/scanner
 

# Screenshots

Token
![image](https://user-images.githubusercontent.com/19740800/219275750-a7f6a944-661b-4b1c-8c1e-83dece153607.png)


NFT
![image](https://user-images.githubusercontent.com/19740800/219271975-df568213-df62-48e8-ba98-152ce1d3f632.png)


Scanner page claim
![image](https://user-images.githubusercontent.com/19740800/219031468-d63d8f51-b3c7-446d-9090-f0fb765180ca.png)


Fixes # (issue)
# Screenshots/Screen Recording of Implementation
